api-docs/tech-ref-info/check-types/agent-check-type-ref.rst

```diff
@@ -188,8 +188,8 @@ Metrics
 +-----------------+--------------------------------------------------+----------+
 | Metric          | Description                                      | Type     |
 +=================+==================================================+==========+
-| avail           | Available space on the filesystem in kilobytes,  | Int64    |
-|                 | including reserved space.                        |          |
```

Comment:
Hi @DonNateR : The reserved space is the space reserved by the system.  Used space is the space used, not including the reserved space.  

Looking at the documentation, I think it assumes that the reader is familiar with these metrics.  I can talk to our engineers to see how we can clarify the documentation further.